action.yaml

```diff
@@ -12,8 +12,30 @@ inputs:
     description: The scan type to perform (k8s or tf)
     required: true
 runs:
-  using: 'docker'
-  image: 'Dockerfile'
+  using: "composite"
+  steps:
+    - name: Install Mondoo Client
+      shell: bash
+      run: |
+          echo ${{ inputs.service_account_credentials }} | base64 -d > mondoo.json
+          curl -sSL https://mondoo.com/install.sh | bash
+    - name: Build the Docker Container
+      shell: bash
+      run: docker build . --file ${{ inputs.path }} --tag mondoo-test-image
```

Comment:
that would mean that users would need to build the image twice. Would it not be easier to pick up the local build image from docker?

Reply:
My 2 cents here are to make it optional step. Add a build parameter to the action and only execute this step if the parameter is true

Reply:
This is basically an optional step at this point. You have to specify docker and have the dockerfile in the path. The plan is to also give you the ability to specify an image name in a future release. The goal of this PR is to allow for a really simple action config if you want to test the result of a Dockerfile. For users without multi-stage build pipelines, their test pipeline is going to be independent of other image build pipelines so there's no real duplication of work. It'll just work for the folks that need something simple out of the box and anyone that needs custom build logic or wants a multi-stage build can use the image directly in the future.

Reply:
Working on a PR that does the image scanning w/o building so it's more clear what the scan types are.